launch/world_modeling/world_model.launch

```diff
@@ -28,7 +28,7 @@
 		</node>
 
 		<node pkg="image_recognition_keras" type="face_properties_node" name="keras_face_properties">
-			<param name="weights_file_path" value="~/src/keras_models/age_gender/weights.28-3.73.hdf5" />
+			<param name="weights_file_path" value="~/data/keras_models/age_gender/weights.18-4.06.hdf5" />
```

Comment:
Are you sure the weights file is correct? Because we just merged the target PR with the `weights.28-3.73.hdf5`

Reply:
Oh, things changed on the release page. The new version isn't available anymore. So only need to change the folder.